apple_sign_in.gemspec

```
@@ -5,8 +5,8 @@ require_relative 'lib/apple_sign_in/base/version'
 Gem::Specification.new do |spec|
   spec.name          = 'apple_sign_in'
   spec.version       = AppleSignIn::Base::VERSION
-  spec.authors       = ['Timothy Peraza']
-  spec.email         = ['timothy@rootstrap.com']
+  spec.authors       = ['Timothy Peraza, Antonieta Alvarez, Martín Moron']
```

```suggestion
  spec.authors       = ['Timothy Peraza, Antonieta Alvarez, Martín Morón']
```

(I know it's silly but otherwise it can be misread as "moron")